### PR TITLE
[DOCS-151] Account Recovery 

### PR DESCRIPTION
**Pull request by ana-dashuk-cobalt**
@molfinn Please review the updates describing the account recovery process.

This is how the changes will appear on the portal:
https://deploy-preview-138--cobalt-docs.netlify.app/getting-started/sign-in/account-recovery/

### CODE REVIEW

File: content/en/Getting started/Sign In/account-recovery.md
Comment (mjang-cobalt) [inline content/en/Getting started/Sign In/account-recovery.md:59]: Info only: While this looks weird (two consecutive punctuation marks), I understand that it matches what's shown in the UI, and conforms to standard grammar.

**Review by mjang-cobalt — APPROVED** [review]
The links work. I'm good with the content too.

**Comment by molfinn** [comment]
My comments: 

Line: On the page prompting you to enter a one-time code, select Start account recovery process. 
• should we make it clear that the "one-time code" here is the 2FA code from mobile app? 

Line: Once you get an email with a recovery code, enter the code, and select Verify.
• it isn't a **recovery code**, it's more like a identity verification code. I don't want the user to confuse this with the maybe more common account recovery code or backup code. We plan to introduce that in the future. 

Line: If you are the only Organization Owner, we’ll turn off 2FA for you.
• should we say, if you're the only Organization owner, Cobalt Staff will turn off 2FA for you. 

Line: You get an email notification confirming that 2FA was turned off.
• The email actually says that your account 2fa settings were updated. we don't currently explicitly say it was turned off. 

Line: You get an email notification confirming that 2FA was turned off for a user. (under Turn Off 2FA for a User)
• The org owner will not get an email saying that 2FA was turned off for a user. Only that user who's 2FA was turned off will receive an email. 
• can we also include something here like we have verified the user's identity with a one-time identity verification code, but we also suggest verifying the user's identity before turning off 2FA according to your company policies.

File: content/en/Getting started/Sign In/account-recovery.md
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Sign In/account-recovery.md:27]: @molfinn The most important in this step is the action: `select **Start account recovery process**`. The first part of the sentence is only needed to understand on which page we should click the link, so there is no need to over-explain this.

File: content/en/Getting started/Sign In/account-recovery.md
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Sign In/account-recovery.md:29]: @molfinn Changed to `one-tme verification code`.

File: content/en/Getting started/Sign In/account-recovery.md
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Sign In/account-recovery.md:32]: @molfinn "We" means Cobalt. It's sufficient for the user.

File: content/en/Getting started/Sign In/account-recovery.md
Comment (ana-dashuk-cobalt) [inline content/en/Getting started/Sign In/account-recovery.md:34]: @molfinn Changed this: `that your 2FA settings were updated`.

**Comment by ana-dashuk-cobalt** [comment]
> @ana-dashuk-cobalt looks good to me. Thanks for addressing

@molfinn Please approve the request if you are OK with the changes. I cannot proceed until all reviewers approve the PR.

**Review by molfinn — APPROVED** [review]

**Pull request merged**
